webfe/package_vue/src/language/lang/en.js

```diff
@@ -1,2163 +0,0 @@
-  '1. 在新版开发者中心部署应用，并确认应用功能都正常': '1. Deploy the APP in the new Developer Centre and make sure it all functions correctly',
-  在新版开发者中心确认应用第三方地址是否正确: 'Confirm that the APP"s third- party address is correct in the new developer center',
-  访问应用: 'Access APP',
-  现在去部署: 'Go Deploy Now',
-  '已下架的应用桌面无法查看，重新部署生产环境可上架到桌面。': 'Archived APP is inaccessible, redeploying prod env will publish it to the Market.',
-  确认迁移: 'Confirm migration',
-  '“确认迁移”': '“Confirm migration”',
-  '将会停掉旧版本中的服务进程，并切换应用访问入口。': 'Will stop the service process in the old version and switch the APP access entry.',
-  应用在旧版开发者中心的测试环境和正式环境都未下架: 'The APP was not archived in both the stag and prod environments of the old developer center',
-  应用在旧版开发者中心的正式环境未下架: 'The APP is not archived in the prod env of the old developer center',
-  应用在旧版开发者中心的测试环境未下架: 'The APP is not archived in the stag env of the old  developer center',
-  现在去下架: 'Go to archive now',
-  请先部署应用并确认功能正常: 'Please deploy the APP first and make sure it works',
-  '经测试功能不正确，取消本次迁移': 'Not normal, cancel this migration',
-  应用预发布环境和生产环境都未下架: 'APP not archived in both the stag and prod envs',
-  应用生产环境未下架: 'APP not archived in prod env',
-  开发语言: 'Language',
-  迁移状态: 'Migration status',
-  已上架: 'On the shelf',
-  未上架: 'Not on the shelf',
-  未启动: 'Not started',
-  迁移到: 'Migration to',
-  新版开发者中心: 'New Developer Center',
-  蓝鲸开发者中心: 'BlueKing Developer Center',
-  查看进度: 'View progress',
-  迁移时间: 'Migration time',
-  回滚至旧版本: 'Rollback to older versions',
-  原因: 'Reasons',
-  迁移应用: 'Migrating APP',
-  '开始前，请确认应用': 'Before you start, please confirm the APP',
-  '我们将单独处理您的迁移请求。': 'We will process your migration request separately.',
-  应用没有访问某些添加了IP白名单的服务: 'The APP does not have access to certain services to which IP whitelisting has been added',
-  '应用迁移到新版开发者中心后，出口IP将会发生变化': 'The egress IP will change when the APP is migrated',
-  '应用没有使用外部 DB': 'The APP is not using an external DB',
-  '如果使用外部DB, 需要根据': 'If using an external DB, need to follow the guidelines in',
-  迁移文档: 'migration doc',
-  的指引操作: '',
-  '应用没有使用 Stackless Python 的相关特性': 'The APP does not use features related to Stackless Python',
-  '应用没有使用NFS服务（之前的文件上传采用的NFS）': 'The APP does not use NFS services',
-  '应用如果对外提供了接口，需要通知调用方修改接口url': 'The APP needs to notify the caller to modify the interface url if the interface is provided externally',
-  '没有使用api gateway，需要通知所有使用到接口的应用或服务 修改url': 'No api gateway is used, all APP or services that use the interface need to be notified to modify the url',
-  '使用了api gateway, 只需要再api gateway 上修改url即可': 'api gateway is used, just change the url on the api gateway',
-  '发给其他人或第三方系统的回调链接需要修改，比如邮件中的链接，权限审批链接等': 'Callback links to other people or third party systems need to be modified, e.g. links in emails, permission approval links, etc.',
-  已阅读: 'Read',
-  '点击【开启迁移】后，你的应用服务将不会受到任何影响': 'Your APP services will not be affected in any way after clicking [Enable Migration]',
```

Comment:
Your APP services will not be affected in any way after clicking "Enable Migration"